client/styles/components/_editor.scss

```diff
@@ -121,7 +121,7 @@ pre.CodeMirror-line {
   height: 100%;
 }
 
-.CodeMirror-find-div {
+.CodMirror-search-inputs {
```

Comment:
i'm assuming you mean for this class to be `.CodeMirror-search-inputs`? I don't see anything in the HTML that has `.CodMirror-search-inputs`, so maybe you can just delete this CSS?

Reply:
I'm sorry. A mistake on my side.

Reply:
no worries!!